ofl/kanchenjunga/article/ARTICLE.en_us.html

```diff
@@ -0,0 +1 @@
+N/A
```

Comment:
Article/Description text is needed.

Reply:
@emmamarichal I saw you marked this as resolved, but the font still needs the Article.

Reply:
Ah? I added it two weeks ago, it doesn't show up on your side?

Reply:
It does now!